### PR TITLE
core:  Rebirth of the explorer

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- Redo the explorer page
- Refactor html + CSS + JS
- Break down CSS + remove inline
- Enable filter by multiple options under the same category
- Update as per design 

##### Checklist

- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

- css
- html
- js

##### Refers
- https://github.com/gitcoinco/web/issues/503
- https://github.com/gitcoinco/web/issues/412
- https://github.com/gitcoinco/web/issues/494

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Waiting for merge from https://github.com/gitcoinco/web/pull/514 to reuse styles

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -52,6 +52,27 @@ var set_filter_header = function() {
   $("#filter").html(filter_status);
 }
 
+// TODO: Refactor function :
+// Deselect option 'any' when another filter is selected
+// Selects option 'any' when no filter is applied
+// TODO : Disable other filters when 'any' is selected
+var disableAny = function () {
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/dashboard.js]: @eswarasai could you refactor this whenever you get to it ?  This is super messy :P

**Comment by owocki** [comment]
this is 🔥 🔥 hot hot hot... so excited to get this shipped

cc @mbeacom for code review.. i will take a look later today too

File: app/assets/v2/js/pages/dashboard.js (hunk 1 above)
Comment (eswarasai) [reply app/assets/v2/js/pages/dashboard.js]: ^Aye cap'n 🖖

**Comment by thelostone-mc** [comment]
## Sneak-Peak

help

@KennethAshley you might have to lend me a hand with the radio + form + button changes (what class to put in and where  ^_^ )

**Comment by mbeacom** [comment]
Holding off on reviewing this until it's out of `WIP` and #514 is merged.

**Comment by mkosowsk** [comment]
@thelostone-mc Looks awesome dude, great work as always! 😍🖖🏻

Quick question for ya, what do you prefer to use to make GIFs for stuff like this?

**Comment by thelostone-mc** [comment]
@mkosowsk Oh I use [LICEcap](https://github.com/justinfrankel/licecap). It's neat and simple :D

**Comment by mbeacom** [comment]
@thelostone-mc #514 has been merged.

**Comment by thelostone-mc** [comment]
@mbeacom Ack and updated PR 
Also I've tested it to a good amount. Since this is sort of the heart of the app -> 
double / triple check would make feel better  :P 

There are a few things which aren't handled in this PR but will be taken care by @eswarasai in a different PR

File: app/dashboard/templates/dashboard.html [hunk 2]
```diff
@@ -30,20 +32,24 @@
...
-            <div class="col-10 ml-3 font-bigger-2" style="color: #3e24fb;padding-left: 1rem;">
-              <span id="modifiers">Funded Issues</span>
-              <span id=stats class="font-bigger-1 pl-1"></span>
+            <div class="col-12 offset-md-1 col-md-10 title-row">
+              <div class="col-4">
+                <span id="filter" class="font-title"></span>
+              </div>
+              <div class="col-8 bounty-info">
+                <span id="modifiers">Funded Issues</span>
+                <span id=stats class="font-body"></span>
```
Comment (eswarasai) [inline app/dashboard/templates/dashboard.html]: @thelostone-mc -- Shouldn't `id` here be wrapped in double quotes?

Comment (thelostone-mc) [reply app/dashboard/templates/dashboard.html]: Oh shoot :P thanks

**Comment by owocki** [comment]
oh 😍 😍 😍 😍 this looks so good!

one thing i dont see (maybe it was missed in the design stage.. let me nkow) is the ability to sort [by these fields](http://bits.owocki.com/0D2a22002Q2q/Screen%20Shot%202018-03-06%20at%209.06.36%20AM.png).  perhaps im just missing that?

**Comment by owocki** [comment]
thanks for testing mobile/tablet!

also just checking -- does `save search` button still work?  i am AFK now or would test myself

**Comment by mbeacom** [comment]
@owocki Testing this now.

**Comment by mbeacom** [comment]
@owocki Good call! The save_search call isn't passing `email` anymore.
@thelostone-mc Also, when selecting a network, it seems a bit confusing to not allow them to directly select `Any`.  What's everyone else think?
network_toggle

**Comment by thelostone-mc** [comment]
@owocki Ah It was removed in the [design](https://camo.githubusercontent.com/fe523cc0abec766bee8eade4fb93f820ee0f27be/68747470733a2f2f776166666c65696f2d6469726563742d75706c6f6164732d70726f64756374696f6e2e73332e616d617a6f6e6177732e636f6d2f75706c6f6164732f3561383332393561613563633637303031333662346263382f31323535313663363665383263373238616365323165306434366366396438653230386438616137623063636137353361383532653239313362303436623331333931366163323137353836663736616535303032653030353930653464666131323563356534376664666464633761613662363338323438613064306462386439336234643638643437663430663265636464343265363462656636323939343539376237623438303464633565393334323265373832386334383363316230633161396262322e706e67) 

will fix up the `save_search` now

**Comment by mbeacom** [comment]
Thoughts on the change of font in the navbar under `Issuer Explorer`?

font_oddity

@PixelantDesign @thelostone-mc

**Comment by owocki** [comment]
i think the fonts should likely stay the same... but broadly defer to @PixelantDesign on this question

**Comment by PixelantDesign** [comment]
Would prefer if the fonts did not jump due to size changes @thelost-one could you take a look? 

We do have an issue that the About link does not show in certain sections, though I think this is a different problem.

**Comment by thelostone-mc** [comment]
Had conflicts to resolve 
Rebasing the code -> I see that the left-rails has been disabled for mobile ,
do we still want that  ?

**Comment by thelostone-mc** [comment]
@mbeacom fixed that up! navbar uses the old font ^_^
I sort of did the mistake of focusing only on the explorer page 😓
Should have kept the bigger picture in mind

**Comment by owocki** [comment]
followup post https://github.com/gitcoinco/web/issues/555

**Pull request merged**